manifests/d/Docker/DockerDesktop/4.1.0.69386/Docker.DockerDesktop.installer.yaml

```
@@ -0,0 +1,25 @@
+# Automatically updated by the winget bot at 2021/Sep/30
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Docker.DockerDesktop
+PackageVersion: 4.1.0.69386
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: exe
+  Scope: machine
+  InstallerUrl: https://desktop.docker.com/win/main/amd64/Docker%20Desktop%20Installer.exe

```

```suggestion
  InstallerUrl: https://desktop.docker.com/win/main/amd64/69386/Docker%20Desktop%20Installer.exe
```

We should be using the versioned URL for this. I’d push a update myself but I’m on mobile atm.